src/test/ui/impl-trait/bound-normalization-pass.rs

```diff
@@ -0,0 +1,77 @@
+// compile-pass
```

Comment:
FYI the `compile-pass` annotation will be removed so we should use `run-pass` or something else.

Reply:
It should probably be `// check-pass`.